### DISCUSSION TITLE
Migrating data between environments

### DISCUSSION THREAD

**Discussion by jameswoodley** [OrchardCMS/OrchardCore#11641] [Q&A] [answered]
Does anyone have any best practice on how to migrate data between environments? We currently have the typical dev -> staging -> prod set up and we are finding issues in prod because the data is different.

I can of course just blanket overwrite the dev/staging environment databases, but then we lost valuable test data as well. Is there a process in Orchard to copy certain content to other Orchard instances?

**Comment by hishamco** [comment, marked as answer]
Try to use Deployment features for import / export

**Comment by devlife** [comment]
To build on @hishamco's answer, I create a deployment plan for each set of related changes. Then execute the deployment plan to push the changes to each environment.
